src/components/blocked-patron/BlockedPatron.test.ts

```diff
@@ -0,0 +1,77 @@
+import { TOKEN_LIBRARY_KEY } from "../../core/token";
+
+describe("Patron page", () => {
+  before(() => {
+    cy.window().then((win) => {
+      win.sessionStorage.setItem(TOKEN_LIBRARY_KEY, "random-token");
```

Comment:
Think we should have a Cypress cmd for creating a library token, like we have for user tokens.